lightkurve/targetpixelfile.py

```diff
@@ -816,6 +818,70 @@ def to_fits(self, output_fn=None, overwrite=False):
             output_fn = "{}-targ.fits".format(self.keplerid)
         self.hdu.writeto(output_fn, overwrite=overwrite, checksum=True)
 
+    def get_hdu(image, extension):
+        """Checks if an image is a FITS single image, or a FITS file,
+        and returns the appropriate PrimaryHDU.
+
+        Attributes
+        ----------
+        image : fits.ImageHDU or fits.HDUList object
+            FITS filename path or ImageHDU object to get the data from.
+        extension: integer 0, or 1
+            Determines which file extension the data is in, depending on the file
+            type and source
+        Returns
+        -------
+        hdu : astropy.PrimaryHDU
+            PrimaryHDU of the image inputted
+        """
+        if isinstance(image, fits.ImageHDU):
+            hdu = image
+        elif isinstance(image, fits.HDUList):
+            hdu = image[extension]
+        else:
+            hdu = fits.open(image)[extension]
+        return hdu
+
+    def get_WCS(mid_img,hdu,position,extension): #name of this?? add doc string
```

Comment:
Same here, except get_WCS is quite close to get_coordinates and WCS which are two other functions. Let's think of another name.